packages/format-library/src/link/inline.js

```diff
@@ -168,20 +150,71 @@ class InlineLinkUI extends Component {
 		this.setState( { inputValue } );
 	}
 
-	setLinkTarget( opensInNewWindow ) {
-		const { activeAttributes: { url = '' }, value, onChange } = this.props;
-
-		this.setState( { opensInNewWindow } );
+	setLinkAttributes( attributes ) {
+		this.setState( {
+			...this.state,
```

Comment:
You don't have to do this with `setState`. `setState` will already keep the previous state attributes around.

Reply:
Ah ofcourse. My mistake :).